### PR TITLE
using reactivation keys / link chain

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Using reactivation key with bootstrap script.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links
Fixes https://github.com/SUSE/spacewalk/issues/13320

### CODE REVIEW

**Review by mcalmer — CHANGES_REQUESTED** [review]
I would also write quickly how to use re-activation keys with salt.
It is about setting a grain on the minion in /etc/salt/minion.d/susemanager.conf.

```
grains:
    susemanager:
        management_key: <reactivation-key>
```

**Comment by keichwa** [comment]
> I would also write quickly how to use re-activation keys with salt.

thanks, this is a good idea.  Please recheck!

File: modules/client-configuration/pages/activation-keys.adoc [hunk 1]
```diff
@@ -99,11 +99,21 @@ Click btn:[Generate New Key] to create the reactivation key.
...
 
-When you have created a reactivation key, you can use it with the [command]``rhnreg_ks`` command line utility.
-This command re-registers the client and restore its {productname} settings.
-You can only use a reactivation key once.
+When you have created a reactivation key, you can use it with a bootstrap script, with Salt, or the [command]``rhnreg_ks`` command line utility.
 
-You can combine reactivation keys with activation keys to aggregate the settings of multiple keys for a single system profile.
+For more information about bootstrap scripts, see xref:client-configuration:registration-bootstrap.adoc[].
+
+With Salt, you set a grain on the Salt client in [path]``/etc/salt/minion.d/susemanager.conf``:
```
Comment (Loquacity) [inline modules/client-configuration/pages/activation-keys.adoc:106]:
```suggestion
For Salt clients, set a grain in [path]``/etc/salt/minion.d/susemanager.conf``:
```

File: modules/client-configuration/pages/activation-keys.adoc [hunk 2]
```diff
@@ -99,11 +99,21 @@ Click btn:[Generate New Key] to create the reactivation key.
...
+
+----
+grains:
+    susemanager:
+        management_key: <reactivation-key>
+----
+
+The [command]``rhnreg_ks`` command is only available for traditional clients.
+It re-registers the client and restores its {productname} settings.
+Only on traditional clients you can combine reactivation keys with activation keys to aggregate the settings of multiple keys for a single system profile.
```
Comment (Loquacity) [inline modules/client-configuration/pages/activation-keys.adoc:116]:
```suggestion
On traditional clients, you can combine reactivation keys with activation keys to aggregate the settings of multiple keys for a single system profile.
```

File: modules/client-configuration/pages/activation-keys.adoc (hunk 1 above)
Comment (Loquacity) [inline modules/client-configuration/pages/activation-keys.adoc]:
```suggestion
For Salt clients, set a grain in [path]``/etc/salt/minion.d/susemanager.conf``:
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Review by Loquacity — APPROVED** [review]

**Comment by keichwa** [comment]
I resolved a conflict.  I hope now nothing is lost nor duplicated.

File: modules/client-configuration/pages/activation-keys.adoc [hunk 3]
```diff
@@ -99,20 +99,21 @@ Click btn:[Generate New Key] to create the reactivation key.
 Record the details of the key for later use.
 Unlike typical activation keys, which are not associated with a specific system ID, keys created here do not show up on the menu:Systems[Activation Keys] page.
 
-When you have created a reactivation key, you can use it with a bootstrap script, with Salt, or the [command]``rhnreg_ks`` command line utility.
-
-For more information about bootstrap scripts, see xref:client-configuration:registration-bootstrap.adoc[].
-
-For Salt clients, set a grain in [path]``/etc/salt/minion.d/susemanager.conf``:
+For Salt clients, after you have created a reactivation key, you can use it as the ``management_key`` grain in [path]``/etc/salt/minion.d/susemanager.conf``. For example:
```
Comment (Loquacity) [inline modules/client-configuration/pages/activation-keys.adoc]:
```suggestion
For Salt clients, after you have created a reactivation key, you can use it as the ``management_key`` grain in [path]``/etc/salt/minion.d/susemanager.conf``. 
For example:
```

File: modules/client-configuration/pages/activation-keys.adoc [hunk 4]
```diff
@@ -99,20 +99,21 @@ Click btn:[Generate New Key] to create the reactivation key.
...
 ----
 grains:
     susemanager:
-        management_key: <reactivation-key>
+        management_key: "re-1-daf44db90c0853edbb5db03f2b37986e"
 ----
 
-The [command]``rhnreg_ks`` command is only available for traditional clients.
-It re-registers the client and restores its {productname} settings.
+You can now restart the [command]``salt-minion`` to apply the reactivation key.
```
Comment (Loquacity) [inline modules/client-configuration/pages/activation-keys.adoc:110]:
```suggestion
Restart the [command]``salt-minion`` process to apply the reactivation key.
```

??

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/activation-keys.adoc (hunk 3 above)
Comment (keichwa) [reply modules/client-configuration/pages/activation-keys.adoc]: by the time, this is already in.

**Review by Loquacity — APPROVED** [review]

**Pull request merged**